lib/SIL/Verifier/SILOwnershipVerifier.cpp

```diff
@@ -623,6 +625,38 @@ bool SILValueOwnershipChecker::isSubobjectProjectionWithLifetimeEndingUses(
   });
 }
 
+bool SILValueOwnershipChecker::
```

Comment:
Thanks for this verification. If we had a reborrow phi flag, then then we wouldn't need any transitive verification at all.